QualityControl/public/layout/Layout.js

```diff
@@ -61,7 +61,7 @@ export default class Layout extends Observable {
     this.cellHeight = 100 / this.gridListSize * 0.95; // %, put some margin at bottom to see below
     this.cellWidth = 100 / this.gridListSize; // %
     // GridList.grid.length: integer, number of rows
-    this.filterModel = model.filterModel;
+    this.applyTreeFilter = true;
```

Comment:
Same for the naming, here something like `shouldApplyTreeFilter` improves readability